OpenEXR/IlmImfTest/testOptimizedInterleavePatterns.cpp

```diff
@@ -167,6 +167,14 @@ Schema Schemes[] =
         {NULL,NULL,NULL,0,NULL,NULL}
 };
 
+template<class T> inline T alignToFour(T input)
```

Comment:
Can bring the cost to basically nothing by doing this
return (input + 3) & ~3

Reply:
OK, I'll update that. I'd not done it that way because I wasn't convinced that would work with 64 bit addresses

Reply:
Right, might need ~3 for 32 bit and ~3ll for 64

Reply:
I'm not really sure the best way to do robust compile-time detection of 32 vs 64 bit architectures.
Given that this is only in a test, not the main library, and is only called a handful of times, I wonder if it might be safer to leave it as it is?

Reply:
oh, I would have thought the 3s would be promoted to the integer type, you could be pedantic and say
return reinterpret_cast<T>( (static_cast<uintptr_t>(input) + static_cast<uintptr_t>(3)) & ~static_cast<uintptr_t>(3) );